Userland/Applications/Presenter/Presentation.cpp

```diff
@@ -43,12 +43,16 @@ StringView Presentation::author() const
 
 bool Presentation::has_a_next_frame() const
 {
-    return m_current_slide < u32(m_slides.size() > 1 ? m_slides.size() - 1 : 0);
+    if (m_current_slide < u32(m_slides.size() > 1 ? m_slides.size() - 1 : 0))
```

Comment:
not really readable; consider extracting the value on the right, e.g. with a name `last_slide`?

(i feel that this condition could be re-written for a better effect but it seems very local, so should not be a problem…)

Reply:
(one of the options for this condition: use `<=` instead — you should not need conditional operator then… 🤔)